tsconfig.json

```diff
@@ -22,6 +24,9 @@
     "noImplicitReturns": true,
     "skipLibCheck": true,
     "moduleResolution": "node",
-    "sourceMap": false
+    "sourceMap": false,
+    "allowJs": true,
+    "checkJs": false,
+    "jsx": "react-jsx"
```

Comment:
@matheushdsbr I believe this configuration "jsx": "react-jsx" should be added as well to your PR.